### PR TITLE
Change url in communication with DNSaaS and add message after save

### PR DESCRIPTION
**Pull request by szok**
- Change DNSaaS urls
- Add message after user save or delete record
- Add build_url method to generate urls for DNSaas

### CODE REVIEW

File: src/ralph/dns/dnsaas.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def get_dns_records(self, ipaddresses):
         ipaddresses = [('ip', i) for i in ipaddresses]
         url = urljoin(
             settings.DNSAAS_URL,
-            'api/records/?{}'.format(
+            'api/v2/records/?{}'.format(
```
Comment (ar4s) [inline src/ralph/dns/dnsaas.py]: can you extract version to variable?

Comment (ar4s) [inline src/ralph/dns/dnsaas.py]: or create some function to build url - some like this:
`
def build_url(resource_name, id=None, version='v2', **get_kwargs)
`

Comment (szok) [inline src/ralph/dns/dnsaas.py]: 👍

File: src/ralph/dns/views.py [hunk 2]
```diff
@@ -71,14 +73,26 @@ def post(self, request, *args, **kwargs):
...
+                    messages.success(
+                        request, _('DNS record has been deleted.')
+                    )
             elif posted_form.cleaned_data.get('pk'):
                 errors = self.dnsaas.update_dns_record(
                     posted_form.cleaned_data
                 )
+                if not errors:
+                    messages.success(
+                        request, _('DNS record has been saved.')
```
Comment (mkurek) [inline src/ralph/dns/views.py]: if you use deleted/created in other cases, then there `updated` suits better than `saved`.

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**